### PR TITLE
update program links

### LINKED ISSUE #1085: Replace program links with links to the program Instagram account.
With the move to the new Berea website all of the program links we have in the application broke. Since not all the programs have their own pages in the new website, we now want to link out to the programs social page. 

All insta accounts: https://www.instagram.com/p/CxwGHLTJMgS/?utm_source=ig_web_copy_link&igshid=MzRlODBiNWFlZA%3D%3D

Additionally, when the "More information" link is clicked, open it in a new tab, instead of replacing the current page with the program's social media link. I may also recommend instead of "More information" we specify what kind of link we are sending them to.

### PR DESCRIPTION
**Pull request by qasema**
## Issue
Fixes #1085 

The links on the landing page for program cards were broken and the programs now want links to be also directing to Instagram and Facebook links if they exist.

## Changes
- Replaced broken links with working links on the new website
- Removed the "Read more" modal on the landing page cards
- Added a footer which includes the Instagram and Facebook logos for respective programs.
- Modified the program model to have a `bereaUrl`, `facebookUrl`, and `instagramUrl` field.
- Made the "Learn more" links open new tabs on the service transcript.

## Testing
- Switch branch to update_program_links
- `source setup.sh` and `./database/reset_database.sh test`
- Run the application and view the landing page.
- Verify all the links work and that they all direct to the correct pages.
- Make sure links on the service transcript work.

### CODE REVIEW

**Review by AndersonStettner — CHANGES_REQUESTED** [review]
a few programs that still link out to the Berea website.

**Review by AndersonStettner — CHANGES_REQUESTED** [review]
In the program cards on the landing page, we have a "Learn More" hyper link, we should remove "Learn More" and just make them into links to the socials.

**Comment by qasema** [comment]
> a few programs that still link out to the Berea website.
some programs do not have Instagram pages so we kept the links that still work. Shall switch those to the General Instagram account for Celts

**Comment by stevensonmichel** [comment]
Regarding the "Learn More" hyperlinks ,it will be a good idea to have one that directs to the main Celts website in the Celts Sponsored Events card, as you did for the programs that don't have a page on the new website.

**Comment by stevensonmichel** [comment]
Also, the social media icons or images seem to not be displayed. I suspect that those images are not in the directory that you provided in the landingPage.html file.

File: database/test_data.py [hunk 1]
```diff
@@ -472,7 +490,9 @@
     {
         "id": 10,
         "programName": "Berea Tutoring",
-        "programUrl": "https://www.berea.edu/celts/community-service-programs/volunteer-opportunities/berea-tutoring-program/",
+        "instagramUrl": "https://www.instagram.com/bereatutoring/",
+        "facebookUrl": "https://www.facebook.com/CELTSbereatutoring",
+        "bereaUrl": "https://www.berea.edu/centers/center-for-excellence-in-learning-through-service/programs/berea-tutoring",
```
Comment (stevensonmichel) [inline database/test_data.py:496]: I am not sure that bereaUrl is a good variable name for those links that direct to program pages or main page of the website.

Comment (ojmakinde) [reply database/test_data.py:496]: Understandable. When I discussed with Anderson yesterday, he felt it was appropriate. Do you have any name suggestions?

Comment (stevensonmichel) [reply database/test_data.py:496]: I would go with "websiteUrl". It is up to you.

**Comment by Karina-Agliullova** [comment]
Facebook link for Berea Buddies is wrong, it takes me to the Adopt a grandparent page. Facebook and Instagram pages for Bonner Scholars are wrong too. Hunger Initiatives' links also take me to the wrong pages. The tests are breaking too.

**Comment by gahimbaref** [comment]
looks good to me:)

**Comment by doucoureb** [comment]
Capture1
The Instagram links of those two programs do not redirect to an Instagram page. It took me to the Bonner Scholar program page

**Review by AndersonStettner — CHANGES_REQUESTED** [review]
Please remove the top footer line and make sure the footers extend for the full width of the card body. 

image

**Review by AndersonStettner — CHANGES_REQUESTED** [review]
If a program does not have a specific social media, it should not have social media links. The "Other CELTS-Sponosred Events' Instagram link needs to be changed to be the actual insta link.

**Comment by stevensonmichel** [comment]
If a program does not have a specific social media link, it is better if there is no icon in its section displaying that social media (Instagram or Facebook). This is the case for Celts-Sponsored Events, Bonner Scholars, and Hunger Initiatives.

**Review by andersoncedu — APPROVED** [review]
This looks good to me.

**Comment by andersoncedu** [comment]
Actually, quick question: is the Bonner scholar program supposed to show for students who aren't Bonner scholars?

**Review by AndersonStettner — APPROVED** [review]

**Pull request merged**